### PR TITLE
chg: load list of roles after successful login. Display information i…

### PR DESCRIPTION
**Pull request by dawidgil**
…n tooltip why create vc is disabled.

### CODE REVIEW

File: src/app/routes/enrolment/enrolment.component.ts [hunk 1]
```diff
@@ -36,7 +38,8 @@ export class EnrolmentComponent implements AfterViewInit {
   };
 
   public isMyEnrolmentShown = false;
-  isExperimentalEnabled$ = this.store.select(SettingsSelectors.isExperimentalEnabled);
+  isCreateVCVisible$ = combineLatest([this.store.select(SettingsSelectors.isExperimentalEnabled), this.store.select(RoleSelectors.hasRoles)])
+    .pipe(map(([isExperimental, hasRoles]) => isExperimental && hasRoles));
```
Comment (jrhender) [inline src/app/routes/enrolment/enrolment.component.ts:42]: I think this is either wrong or the name can be improved. Does `hasRoles` mean that the DID controls some role definitions? Or does it mean that the DID is an issuer of some roles?

**Review by jrhender — CHANGES_REQUESTED** [review]

**Comment by jrhender** [comment]
Also, no so important, but what does `chg` mean in the commit message? Is this a standard commit prefix?

File: src/app/routes/enrolment/enrolment.component.ts (hunk 1 above)
Comment (dawidgil) [reply src/app/routes/enrolment/enrolment.component.ts:42]: hasRole selectors returns true/false depending if the user have some roles inside role management tab inside governance. Probably i should update name for example `issuerHasRoles`

**Comment by dawidgil** [comment]
chg = change. I know i do not always follow conventional commits/ angular commit message format. However most of the time i'm squashing commits into one so those smaller are commits "in progress" and mostly they are for me.

File: src/app/routes/enrolment/enrolment.component.ts (hunk 1 above)
Comment (jrhender) [reply src/app/routes/enrolment/enrolment.component.ts:42]: Ok thanks. I think this behaviour is incorrect then. I don't think is relevant whether or not the DID controls roles, the relevant consideration is whether or not the DID can issue any roles.

Regarding the variable name, I think a better name would be `controlsRoles`

Comment (dawidgil) [reply src/app/routes/enrolment/enrolment.component.ts]: Ok, got it. Now i know that i used wrong method. I think i should use `getAllowedRolesByIssuer` from domains service. I will adjust it in a moment. Thanks!

**Comment by dawidgil** [comment]
Now it should be correct. I'm using `getAllowedRolesByIssuer` to get list of possible roles that user can issue.

File: src/app/routes/enrolment/enrolment.component.html [hunk 2]
```diff
@@ -4,8 +4,10 @@
       <div class="col-xl-12">
         <div class="d-flex flex-column flex-md-row justify-content-between align-items-center">
           <h4 class="color-white strong my-4">Issue Role</h4>
-          <button *ngIf="isExperimentalEnabled$ | async" mat-raised-button
-                  class="btn btn-primary btn-small mb-3 mb-lg-0" (click)="createVC()">Create VC
+          <button *ngIf="(isExperimental$ | async) && controlsRoles" mat-raised-button
```
Comment (jrhender) [inline src/app/routes/enrolment/enrolment.component.html:7]: Sorry about this. I think my previous comment was pretty confusing. `controlsRoles` was the right name for the previous behaviour. But I think now that the behaviour has been corrected, `issuesRoles` is a better name.
```suggestion
          <button *ngIf="(isExperimental$ | async) && issuesRoles" mat-raised-button
```

**Review by jrhender — APPROVED** [review]
Please do the variable rename but then it can be merged. Thanks @dawidgil !

File: src/app/routes/enrolment/enrolment.component.html (hunk 2 above)
Comment (dawidgil) [reply src/app/routes/enrolment/enrolment.component.html]: No problem :) Naming sometimes can be hard or tricky... I adjusted PR, thanks for checking!

**Pull request merged**